substrate/runtime/primitives/src/testing.rs

```diff
@@ -157,12 +157,10 @@ impl<Call: AuxDispatchable + Slicable + Sized + Send + Sync + Serialize + Deseri
 		self.0.encode()
 	}
 }
-impl<Call: 'static + AuxDispatchable + Slicable + Sized + Send + Sync + Serialize + DeserializeOwned + Clone + Eq + Debug> Checkable for TestXt<Call> {
+impl<Call: 'static + AuxDispatchable + Slicable + Sized + Send + Sync + Serialize + DeserializeOwned + Clone + Eq + Debug, Ctx> Checkable<Ctx> for TestXt<Call> {
```

Comment:
are these bounds actually necessary?

Reply:
i'll try to reduce them

Reply:
simplified https://github.com/paritytech/polkadot/pull/287/commits/f4fcebb8d3700a2642560cb7b4e486eecb8d5977